.github/workflows/extra_pr_labels.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check for conflicting PRs
-        uses: eps1lon/actions-label-merge-conflict@releases/2.x
+        uses: eps1lon/actions-label-merge-conflict@513a24fc7dca40990863be2935e059e650728400
```

Comment:
I couldn't figure out why, but bumping the version was necessary and there were no newer releases to use. TG was using a wholly different script.